docs/create/aws/index-aws.mdx

```diff
@@ -56,5 +56,17 @@ The following links provide you with the available options to run apps on Heroku
         page="/create/aws/chatapp"
         />
  </div>
+ </div>
+ <div class="row">
+ <div class="col">
+     <RedisCard
```

Comment:
Is this a link back to the page that the user is already on?

Reply:
No @simonprickett This is just for overview page https://developer.redis.com/create/aws